src/local-state/LocalState.ts

```diff
@@ -198,6 +244,34 @@ export declare namespace LocalState {
   export type Path = Array<string | number>;
 }
 
+/**
+ * LocalState enables the use of `@client` fields in GraphQL queries and mutations.
```

Comment:
```suggestion
 * LocalState enables the use of `@client` fields in GraphQL operations.
```

You can use `@client` fields with subscriptions too, so probably better to shorten this to "operations" instead.